src/main/java/org/commcare/suite/model/Suite.java

```diff
@@ -46,18 +46,20 @@
     private final HashMap<String, List<Menu>> rootToMenus = new HashMap<>();
 
     private Vector<Menu> menus;
+    private Hashtable<String, Endpoint> endpoints;
 
     @SuppressWarnings("unused")
     public Suite() {
 
     }
 
     public Suite(int version, Hashtable<String, Detail> details,
-                 Hashtable<String, Entry> entries, Vector<Menu> menus) {
+                 Hashtable<String, Entry> entries, Vector<Menu> menus, Hashtable<String, Endpoint> endpoints) {
         this.version = version;
         this.details = details;
         this.entries = entries;
         this.menus = menus;
+        this.endpoints = endpoints;
```

Comment:
Should this be `externalized`?

Reply:
yes, totally missed out on it. Thanks for the catch